client/src/components/commonComponents/TabsCheckout/TabsCheckout.jsx

```diff
@@ -47,8 +48,12 @@ const TabsCheckout = () => {
 
   const submitForm = async (values, actions) => {
     await sleep(1000);
-    // alert here will be replaced with API post request to store the data into database when reserving the date
-    alert(JSON.stringify(values, null, 2));
+    // the response will be used to setState for the confirmation alert later.
+    axios
+      .post('/api/questions/user-info', values)
+      .then((res) => res.data)
+      .catch((err) => err.response.data.message);
+
     actions.setSubmitting(false);
```

Comment:
would be nice if you'd be able to put a comment here explaining what `actions.setSubmitting(false);` does

Reply:
Yeah, sure. 👌🏻